### PR TITLE
[TASK] deploy a TYPO3 CMS 7 with web folder

### PR DESCRIPTION
**Pull request by achimfritz**

### CODE REVIEW

File: Classes/Lightwerk/SurfTasks/Task/TYPO3/CMS/ExtbaseCommandTask.php [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+ * Creates the command for an extbase commando
+ *
+ * @package Lightwerk\SurfTasks
+ */
+abstract class ExtbaseCommandTask extends Task {
+
+	/**
+	 * @param Deployment $deployment
+	 * @param Application $application
+	 * @param $extensionName
```
Comment (ervaude) [inline Classes/Lightwerk/SurfTasks/Task/TYPO3/CMS/ExtbaseCommandTask.php]: string

File: Classes/Lightwerk/SurfTasks/Task/TYPO3/CMS/ExtbaseCommandTask.php [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+	 * @param $arguments
```
Comment (ervaude) [inline Classes/Lightwerk/SurfTasks/Task/TYPO3/CMS/ExtbaseCommandTask.php]: string

File: Classes/Lightwerk/SurfTasks/Task/TYPO3/CMS/ExtbaseCommandTask.php [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+	 * @param array $options
+	 * @return void
```
Comment (ervaude) [inline Classes/Lightwerk/SurfTasks/Task/TYPO3/CMS/ExtbaseCommandTask.php]: array

File: Classes/Lightwerk/SurfTasks/Task/TYPO3/CMS/ExtbaseCommandTask.php [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+			}
+			$commands[] = $command;
+		}
+		return $commands;
+	}
+
+	/**
+	 * @param Deployment $deployment
+	 * @param Application $application
+	 * @param $extensionName
```
Comment (ervaude) [inline Classes/Lightwerk/SurfTasks/Task/TYPO3/CMS/ExtbaseCommandTask.php]: string

File: Classes/Lightwerk/SurfTasks/Task/TYPO3/CMS/ExtbaseCommandTask.php [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+	 * @param $arguments
```
Comment (ervaude) [inline Classes/Lightwerk/SurfTasks/Task/TYPO3/CMS/ExtbaseCommandTask.php]: string

File: Classes/Lightwerk/SurfTasks/Task/TYPO3/CMS/ExtbaseCommandTask.php [hunk 6]
```diff
@@ -0,0 +1,76 @@
+<?php
+namespace Lightwerk\SurfTasks\Task\TYPO3\CMS;
+
+/*                                                                        *
+ * This script belongs to the TYPO3 Flow package "TYPO3.Surf".            *
```
Comment (ervaude) [inline Classes/Lightwerk/SurfTasks/Task/TYPO3/CMS/ExtbaseCommandTask.php]: Lightwerk.SurfTasks :)

**Comment by ervaude** [comment]
Very nice. Only nitpicks :)

**Comment by achimfritz** [comment]
done, annotations cleaned

**Comment by ervaude** [comment]
:+1:

**Pull request merged**